java/arcs/core/data/Plan.kt

```diff
@@ -21,7 +21,8 @@ import arcs.core.util.lens
 open class Plan(
     // TODO(cromwellian): add more fields as needed (e.g. RecipeName, etc for debugging)
     val particles: List<Particle>,
-    val arcId: String? = null
+    val arcId: String? = null,
+    val name: String = "name"
```

Comment:
is this name only used by logging? I would make the default string "loggablePlanName" or some such to document the purpose (that its supposed to be safe to log)

Reply:
Renamed to nameForLogging.